# Awesome-iGEM-Models

<table>
<thead>
  <tr>
    <th colspan="6">&nbsp;&nbsp;&nbsp;<br>Projects&nbsp;&nbsp;&nbsp;about CRISPR&nbsp;&nbsp;&nbsp;</th>
  </tr>
</thead>
<tbody>
  <tr>
    <td>&nbsp;&nbsp;&nbsp;<br>Name&nbsp;&nbsp;&nbsp;</td>
    <td>&nbsp;&nbsp;&nbsp;<br>Type&nbsp;&nbsp;&nbsp;</td>
    <td>&nbsp;&nbsp;&nbsp;<br>Functions&nbsp;&nbsp;&nbsp;</td>
    <td>&nbsp;&nbsp;&nbsp;<br>Team&nbsp;&nbsp;&nbsp;</td>
    <td>&nbsp;&nbsp;&nbsp;<br>Link&nbsp;&nbsp;&nbsp;</td>
    <td>&nbsp;&nbsp;&nbsp;<br>Evaluation&nbsp;&nbsp;&nbsp;</td>
  </tr>
  <tr>
    <td>&nbsp;&nbsp;&nbsp;<br>Target Recognition Model&nbsp;&nbsp;&nbsp;</td>
    <td>&nbsp;&nbsp;&nbsp;<br>Kinetic model&nbsp;&nbsp;&nbsp;</td>
    <td>&nbsp;&nbsp;&nbsp;<br>Find the numbers and locations of lure&nbsp;&nbsp;&nbsp;sequences and give coresponding intructions to the project&nbsp;&nbsp;&nbsp;</td>
    <td>&nbsp;&nbsp;&nbsp;<br>SJTU-BioX-Shanghai&nbsp;&nbsp;&nbsp;</td>
    <td>&nbsp;&nbsp;&nbsp;<br>https://2019.igem.org/Team:SJTU-BioX-Shanghai/Model&nbsp;&nbsp;&nbsp;</td>
    <td>&nbsp;&nbsp;&nbsp;<br>The approximation works well for single&nbsp;&nbsp;&nbsp;mismatch situation. A computational solution for multi-mismatch situation is&nbsp;&nbsp;&nbsp;provided.&nbsp;&nbsp;&nbsp;</td>
  </tr>
  <tr>
    <td>&nbsp;&nbsp;&nbsp;<br>1. Regulatory Model<br>&nbsp;&nbsp;&nbsp;<br>2. Stochastic Model of dCas9 Binding and Replication&nbsp;&nbsp;&nbsp;Initiation<br>&nbsp;&nbsp;&nbsp;<br>3. Plasmid Copy Number Hacking Model<br>&nbsp;&nbsp;&nbsp;<br>4. Gene Expression Noise Control Model<br>&nbsp;&nbsp;&nbsp;<br>5. Productivity Model<br>&nbsp;&nbsp;&nbsp;<br>6. Quorum Sensing Model&nbsp;&nbsp;&nbsp;</td>
    <td>&nbsp;&nbsp;&nbsp;<br>1. ODE<br>&nbsp;&nbsp;&nbsp;<br>2. Markov chain<br>&nbsp;&nbsp;&nbsp;<br>3. Logistic process<br>&nbsp;&nbsp;&nbsp;<br>4. Yule-Furry process<br>&nbsp;&nbsp;&nbsp;<br>5. ODE<br>&nbsp;&nbsp;&nbsp;<br>6. PDE&nbsp;&nbsp;&nbsp;</td>
    <td>&nbsp;&nbsp;&nbsp;<br>1. Explain the mechanisms of their&nbsp;&nbsp;&nbsp;system's regulation parts<br>&nbsp;&nbsp;&nbsp;<br>2. Show that their system hacks the&nbsp;&nbsp;&nbsp;bacteria's replication rate<br>&nbsp;&nbsp;&nbsp;<br>3. Discuss how their system can control&nbsp;&nbsp;&nbsp;the plasmid copy number<br>&nbsp;&nbsp;&nbsp;<br>4. Illustrate how their system can&nbsp;&nbsp;&nbsp;control the expression noise in a cell<br>&nbsp;&nbsp;&nbsp;<br>5. Explain why their system is able to&nbsp;&nbsp;&nbsp;increase the cells' productivity to multiple types of products like GFP and&nbsp;&nbsp;&nbsp;indigo<br>&nbsp;&nbsp;&nbsp;<br>6. Illustrate how their system works&nbsp;&nbsp;&nbsp;coupling with a quorum sensing system&nbsp;&nbsp;&nbsp;</td>
    <td>&nbsp;&nbsp;&nbsp;<br>Peking&nbsp;&nbsp;&nbsp;</td>
    <td>&nbsp;&nbsp;&nbsp;<br>https://2019.igem.org/Team:Peking/Model&nbsp;&nbsp;&nbsp;</td>
    <td>&nbsp;&nbsp;&nbsp;<br>5 and 6 were validated by  experiments.&nbsp;&nbsp;&nbsp;</td>
  </tr>
  <tr>
    <td>&nbsp;&nbsp;&nbsp;<br>1. CRISPR SHERLOCK Kinetic Model<br>&nbsp;&nbsp;&nbsp;<br>2. Glutathione Detection Kinetic Model<br>&nbsp;&nbsp;&nbsp;<br>3. Eicosane Detection Kinetic Model&nbsp;&nbsp;&nbsp;</td>
    <td>&nbsp;&nbsp;&nbsp;<br>Kinetic model&nbsp;&nbsp;&nbsp;</td>
    <td>&nbsp;&nbsp;&nbsp;<br>1. Understand how different CPLX1&nbsp;&nbsp;&nbsp;concentrations impact the time until maximum active GFP was established<br>&nbsp;&nbsp;&nbsp;<br>2. Follow glutathione degradation into&nbsp;&nbsp;&nbsp;glycine<br>&nbsp;&nbsp;&nbsp;<br>3. Understand the amount of RFP produced&nbsp;&nbsp;&nbsp;in the presence of varying concentrations of either glutathione or eicosane&nbsp;&nbsp;&nbsp;</td>
    <td>&nbsp;&nbsp;&nbsp;<br>Newcastle&nbsp;&nbsp;&nbsp;</td>
    <td>&nbsp;&nbsp;&nbsp;<br>https://2019.igem.org/Team:Newcastle/Model&nbsp;&nbsp;&nbsp;</td>
    <td>&nbsp;&nbsp;&nbsp;<br>1. The model would give a more true&nbsp;&nbsp;&nbsp;reflection of the system if crRNA degradation by trans-cleavage was also&nbsp;&nbsp;&nbsp;included. Rates used were equal to degradation of CPLX1 mRNA degradation and&nbsp;&nbsp;&nbsp;GFP activation. Inclusion of crRNA degradation did not impact time until&nbsp;&nbsp;&nbsp;maximum GFP concentration.<br>&nbsp;&nbsp;&nbsp;<br>2. The model should be re-parameterised&nbsp;&nbsp;&nbsp;in the future to include enzyme rates for E. coli become available.<br>&nbsp;&nbsp;&nbsp;<br>3. The model allowed greater&nbsp;&nbsp;&nbsp;understanding of the system and careful consideration for the other two&nbsp;&nbsp;&nbsp;systems to ensure both would be functional.&nbsp;&nbsp;&nbsp;</td>
  </tr>
  <tr>
    <td>&nbsp;&nbsp;&nbsp;<br>ASFAST Simulation Model&nbsp;&nbsp;&nbsp;</td>
    <td>&nbsp;&nbsp;&nbsp;<br>Kinetic model&nbsp;&nbsp;&nbsp;</td>
    <td>&nbsp;&nbsp;&nbsp;<br>Find out the reaction rate of ASFAST&nbsp;&nbsp;&nbsp;under different virus DNA concentration and take this as a reference to set&nbsp;&nbsp;&nbsp;its detection limit&nbsp;&nbsp;&nbsp;</td>
    <td>&nbsp;&nbsp;&nbsp;<br>CCU Taiwan&nbsp;&nbsp;&nbsp;</td>
    <td>&nbsp;&nbsp;&nbsp;<br>https://2019.igem.org/Team:CCU_Taiwan/Model&nbsp;&nbsp;&nbsp;</td>
    <td>&nbsp;&nbsp;&nbsp;<br>More researches has to be done on the&nbsp;&nbsp;&nbsp;dissociation rate of trans activity to determine the exact rate of its&nbsp;&nbsp;&nbsp;reverse reaction.&nbsp;&nbsp;&nbsp;</td>
  </tr>
  <tr>
    <td>&nbsp;&nbsp;&nbsp;<br>1. Dynamic Model of Heavy Metal Detection&nbsp;&nbsp;&nbsp;Biosensor<br>&nbsp;&nbsp;&nbsp;<br>2. Free Energy Model of Off-target&nbsp;&nbsp;&nbsp;Problem&nbsp;&nbsp;&nbsp;</td>
    <td>&nbsp;&nbsp;&nbsp;<br>1. ODE<br>&nbsp;&nbsp;&nbsp;<br>2. Probability theory and dynamic&nbsp;&nbsp;&nbsp;deduction&nbsp;&nbsp;&nbsp;</td>
    <td>&nbsp;&nbsp;&nbsp;<br>1. Characterize the pathways&nbsp;&nbsp;&nbsp;quantitatively and predict their performance<br>&nbsp;&nbsp;&nbsp;<br>2. Investigate the off-target problem in&nbsp;&nbsp;&nbsp;gene editing by the CRISPR-Cas system&nbsp;&nbsp;&nbsp;</td>
    <td>&nbsp;&nbsp;&nbsp;<br>TJU China&nbsp;&nbsp;&nbsp;</td>
    <td>&nbsp;&nbsp;&nbsp;<br>http://2018.igem.org/Team:TJU_China/Model&nbsp;&nbsp;&nbsp;</td>
    <td>&nbsp;&nbsp;&nbsp;<br>1. The concentration of smURFP is&nbsp;&nbsp;&nbsp;relatively sensitive to parameters such as ktx3,ktl3,ktx4,kb4,kb6,kd2,kd5,&nbsp;&nbsp;&nbsp;kd6,kd7,kd8,kd11, etc. However, due to the lack of previous modeling studies&nbsp;&nbsp;&nbsp;on dCas9-RNAP, some kinetic parameters may not be very accurate.<br>&nbsp;&nbsp;&nbsp;<br>2. After testing, they find its rate can&nbsp;&nbsp;&nbsp;reach approximately 2*10^8 base/h(under parallel computing in 4cores).&nbsp;&nbsp;&nbsp;Besides the default parameters, they hope their model can hit more true data.&nbsp;&nbsp;&nbsp;</td>
  </tr>
  <tr>
    <td>&nbsp;&nbsp;&nbsp;<br>System Modeling&nbsp;&nbsp;&nbsp;</td>
    <td>&nbsp;&nbsp;&nbsp;<br>ODE&nbsp;&nbsp;&nbsp;</td>
    <td>&nbsp;&nbsp;&nbsp;<br>Predict the time point when the all&nbsp;&nbsp;&nbsp;antibiotic resistance genes(ARGs) are cut off and the cell auto-lysis is&nbsp;&nbsp;&nbsp;going to begin&nbsp;&nbsp;&nbsp;</td>
    <td>&nbsp;&nbsp;&nbsp;<br>ZJUT-China&nbsp;&nbsp;&nbsp;</td>
    <td>&nbsp;&nbsp;&nbsp;<br>http://2018.igem.org/Team:ZJUT-China/Model&nbsp;&nbsp;&nbsp;</td>
    <td>&nbsp;&nbsp;&nbsp;<br>Just a quite weak pulsed light signal is&nbsp;&nbsp;&nbsp;enough to start the operation of their system. The system is safe because the&nbsp;&nbsp;&nbsp;accidental leakage of bacteria will cut off all the ARGs and autolysis for&nbsp;&nbsp;&nbsp;the natural light.&nbsp;&nbsp;&nbsp;</td>
  </tr>
  <tr>
    <td>&nbsp;&nbsp;&nbsp;<br>SpliceMIT – Splice Modelling Intronic&nbsp;&nbsp;&nbsp;Technology&nbsp;&nbsp;&nbsp;</td>
    <td>&nbsp;&nbsp;&nbsp;<br>Non-parametric statistical method…&nbsp;&nbsp;&nbsp;</td>
    <td>&nbsp;&nbsp;&nbsp;<br>Generate antisense oligonucleotides (ASO)&nbsp;&nbsp;&nbsp;for a given DNA/RNA sequence, and then analyze and output the most effective&nbsp;&nbsp;&nbsp;ASOs&nbsp;&nbsp;&nbsp;</td>
    <td>&nbsp;&nbsp;&nbsp;<br>MIT&nbsp;&nbsp;&nbsp;</td>
    <td>&nbsp;&nbsp;&nbsp;<br>http://2017.igem.org/Team:MIT/Model&nbsp;&nbsp;&nbsp;</td>
    <td>&nbsp;&nbsp;&nbsp;<br>The probabilities of secondary structure&nbsp;&nbsp;&nbsp;of gRNA sequence and pre-gRNA sequences are compared. The correlation&nbsp;&nbsp;&nbsp;efficient is 0.91, with an extremely low p-value.&nbsp;&nbsp;&nbsp;&nbsp;</td>
  </tr>
  <tr>
    <td>&nbsp;&nbsp;&nbsp;<br>CRISPR Activity Model&nbsp;&nbsp;&nbsp;</td>
    <td>&nbsp;&nbsp;&nbsp;<br>Biophysical model&nbsp;&nbsp;&nbsp;</td>
    <td>&nbsp;&nbsp;&nbsp;<br>Provide the program with the appropriate&nbsp;&nbsp;&nbsp;genome and target gene sequence, and return a list of potential guide&nbsp;&nbsp;&nbsp;sequences ranked from best to worst&nbsp;&nbsp;&nbsp;</td>
    <td>&nbsp;&nbsp;&nbsp;<br>British Columbia&nbsp;&nbsp;&nbsp;</td>
    <td>&nbsp;&nbsp;&nbsp;<br>http://2017.igem.org/Team:British_Columbia/Model&nbsp;&nbsp;&nbsp;</td>
    <td>&nbsp;&nbsp;&nbsp;<br>Using this model, they have designed&nbsp;&nbsp;&nbsp;sgRNAs to target essential virulence regions in Agrobacterium. The guides&nbsp;&nbsp;&nbsp;obtained from running the model were shown to be effective in vitro.&nbsp;&nbsp;&nbsp;</td>
  </tr>
  <tr>
    <td>&nbsp;&nbsp;&nbsp;<br>LacILov System Model&nbsp;&nbsp;&nbsp;</td>
    <td>&nbsp;&nbsp;&nbsp;<br>ODE&nbsp;&nbsp;&nbsp;</td>
    <td>&nbsp;&nbsp;&nbsp;<br>Capture the dynamics of construct&nbsp;&nbsp;&nbsp;developed in the Wet Lab to further characterize its behaviour&nbsp;&nbsp;&nbsp;</td>
    <td>&nbsp;&nbsp;&nbsp;<br>Toronto&nbsp;&nbsp;&nbsp;</td>
    <td>&nbsp;&nbsp;&nbsp;<br>2017.igem.org/Team:Toronto/Drylab&nbsp;&nbsp;&nbsp;</td>
    <td>&nbsp;&nbsp;&nbsp;<br>Adjusted R-squared=0,8238<br>&nbsp;&nbsp;&nbsp;<br>p-value=9.741e-16&nbsp;&nbsp;&nbsp;</td>
  </tr>
</tbody>
</table>
